ui/jstests/test_manage_taxonomies.jsx

```diff
@@ -32,17 +45,79 @@ define(['QUnit', 'jquery', 'setup_manage_taxonomies', 'reactaddons',
       }
     ]
   };
+  var vocabularies = [
+    {
+      "vocabulary": vocabulary,
+      "terms": vocabulary.terms
+    }
+  ];
+  var saveVocabularyResponse;
```

Comment:
It looks like you could move this and `updateParent` inside the test where they are being used. In general we want to define variables as close to their use as possible